### PR TITLE
Add embedded YouTube recording to docs landing page

### PR DESCRIPTION
**Pull request by mattray**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -27,16 +27,16 @@ If you want more accurate results or Cloud Costs from your provider's bill pleas
 
 The [OpenCost Specification](specification) describes a vendor-neutral implementation for Kubernetes cost monitoring.
 
+## Using the UI
+
+The OpenCost UI provides a visualization of the Kubernetes allocations and the related Cloud Costs. This walkthrough highlights some of the components of the OpenCost UI:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/j_4aaE5zk7M?si=N01C23OyAx3qa12b" title="OpenCost UI walkthrough" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (dwbrown2) [inline docs/index.md:34]: this width is much smaller than main context width. can we make wider?

**Review by dwbrown2 — APPROVED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -27,16 +27,16 @@ If you want more accurate results or Cloud Costs from your provider's bill pleas
...
+
 ## Stay in the Loop
 
 Join the [OpenCost Community](community) and follow along other community events using our [calendar](https://calendar.google.com/calendar/u/0/embed?src=c_c0f7q56e5eeod3j89bb320fvjg@group.calendar.google.com&ctz=America/Los_Angeles).
 
 Join us on [CNCF Slack](https://slack.cncf.io/) in the [#opencost](https://cloud-native.slack.com/archives/C03D56FPD4G) channel if you have questions!
 
 Contact us via email (<opencost@kubecost.com>) if there's anything else you need.
-
```
Comment (dwbrown2) [inline docs/index.md:37]: It's still not clear how to access the UI on this page...

File: docs/index.md (hunk 1 above)
Comment (mattray) [reply docs/index.md]: I made it 150% larger

File: docs/index.md (hunk 2 above)
Comment (mattray) [reply docs/index.md:37]: I added port-forward and access instructions

**Pull request merged**